Bugsnag/Helpers/BSGRunContext.m

```diff
@@ -26,6 +28,7 @@
 
 #pragma mark Forward declarations
 
+static uint64_t BSGRunContextGetBootTime(void);
```

Comment:
Style nit: I feel that prefixing static functions in the same way as public functions makes it harder to tell which is API and which is not. We don't need to artificially namespace these since their names are not exported.

Reply:
https://github.com/bugsnag/bugsnag-cocoa/pull/1342